stl/inc/format

```diff
@@ -722,8 +722,11 @@ public:
         }
     };
 
-    // TRANSITION, LLVM-49072
+#if defined(__clang__) || defined(__EDG__) // TRANSITION, LLVM-81774 (Clang), VSO-1956558 (EDG)

```

Comment:
This is _not_ removing a bug workaround. You had one job! (No change requested.)